modules/ztp-image-based-upgrade-procedure-rollback.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-rollback_{context}"]
+= Rolling back an image-based upgrade on managed clusters at scale
+
+Roll back the changes on a set of managed clusters if you encounter unresolvable issues after an upgrade.
```

Comment:
In ibgu, rollbacks only is used for rolling back successful upgrades. for failed upgrades we rely on auto rollback which will transition the ibu status to upgrade-failed. Then user can use `abortOnFailure` or `abort` to go back to `idle`.